src/index.html

```diff
@@ -1,13 +1,51 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/index.css">
+  </head>
+  <body>
+    <article class="card" data-qa="card">
+      <img 
+        src="./images/imac.jpeg" 
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        class="card__image"
+      >
+
+      <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+      <p class="card__code">Product code: 195434</p>
+      
+      <div class="card__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="card__reviews">Reviews: 5</p>
+      </div>
+
+      <div class="card__price">
+        <h3 class="card__price-label">Price:</h3>
+        <p class="card__price-value">$2,199</p>
+      </div>
+  
+      <a 
+        href="#buy" 
+        class="card__buy-link" 
+        data-qa="hover"
+      >
+          Buy
+      </a>
```

Comment:
```suggestion
      >
        Buy
      </a>
```
look like there are 4 indentations here)